src/test/e2e/00_use_cli_test.go

```diff
@@ -86,18 +86,18 @@ func TestUseCLI(t *testing.T) {
 		controllerImageWithTag := "quay.io/jetstack/cert-manager-controller:v1.11.1"
 		controlImageWithSignature := "quay.io/jetstack/cert-manager-controller:sha256-4f1782c8316f34aae6b9ab823c3e6b7e6e4d92ec5dac21de6a17c3da44c364f1.sig"
 
-		// Test `zarf prepare find-images` on a chart that has a `kubeVersion` declaration greater than the default (v1.20.0)
+		// Test `zarf prepare find-images` on a chart that has a `kubeVersion` declaration greater than the previous default (v1.20.0)
+		// We now default to the kubeVersion value from the kubernetes library so we should be up to date
```

Comment:
```suggestion
		// Test `zarf prepare find-images` on a chart that has a `kubeVersion` declaration greater than the Helm default (v1.20.0)
		// This should pass because we build Zarf specifying the kubeVersion value from the kubernetes client-go library instead
```